Classes/TYPO3/Form/Finishers/RedirectFinisher.php

```diff
@@ -80,4 +80,28 @@ public function setOptions(array $options)
     {
         $this->options = $options;
     }
+
+    /**
+     * @param $request
+     * @return string
+     * @throws \TYPO3\Flow\Mvc\Routing\Exception\MissingActionNameException
```

Comment:
would remove this since it's inherited and thus is not useful and hard to maintain